ntoskrnl/inbv/inbv.c

```diff
@@ -137,52 +159,50 @@ BootLogoFadeIn(VOID)
     UCHAR PaletteBitmapBuffer[sizeof(BITMAPINFOHEADER) + sizeof(MainPalette)];
     PBITMAPINFOHEADER PaletteBitmap = (PBITMAPINFOHEADER)PaletteBitmapBuffer;
     LPRGBQUAD Palette = (LPRGBQUAD)(PaletteBitmapBuffer + sizeof(BITMAPINFOHEADER));
-
     ULONG Iteration, Index, ClrUsed;
 
-    /* Check if we're installed and we own it */
-    if (InbvBootDriverInstalled &&
-        (InbvDisplayState == INBV_DISPLAY_STATE_OWNED))
-    {
-        /* Acquire the lock */
-        InbvAcquireLock();
+    LARGE_INTEGER Delay; // Specified in nanoseconds
```

Comment:
Not nanoseconds

Reply:
Heh indeed, it's in hundreds of nanoseconds :) I'll fix the comments; the calculations/values used for Delay are however OK.